staging/src/k8s.io/apiserver/pkg/cel/library/libraries.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package library
+
+import "github.com/google/cel-go/cel"
+
+// RegisteredLibraries provides the list of all registered CEL libraries.
+func RegisteredLibraries() []RegisteredLibrary {
+	return []RegisteredLibrary{
+		{Name: "cel.lib.ext.strings"},
```

Comment:
can/should we gather decls from the CEL strings library?

Reply:
They're not exported...  I don't know if we need this change at all per the https://github.com/kubernetes/kubernetes/pull/120060#discussion_r1303029869 suggestion. I'll remove.